src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -319,37 +350,82 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
       });
   }
 
+  private countriesEqual(a: SkyCountryFieldCountry, b: SkyCountryFieldCountry) {
+    return a.iso2 === b.iso2 && a.name === b.name;
+  }
+
   private removeEventListeners(): void {
     this.idle.next();
     this.idle.complete();
   }
 
+  private setupCountries(): void {
+    /**
+     * The json functions here ensures that we get a copy of the array and not the global original.
+     * This ensures that multiple instances of the component don't overwrite the original data.
+     *
+     * We must type the window object as any here as the intl-tel-input library adds its object
+     * to the main window object.
+     */
+    this.countries = JSON
+      .parse(JSON.stringify((window as any)
+        .intlTelInputGlobals
+        .getCountryData()));
+
+    this.isInPhoneField = (<HTMLElement>this.elRef.nativeElement.parentElement)
+      .classList
+      .contains('sky-phone-field-country-search');
+
+    /* istanbul ignore else */
+    if (!this.isInPhoneField) {
+      /**
+       * The library we get the country data from includes extra phone properties.
+       * We want to remove these unless we are in a phone field
+       */
+      this.countries.forEach((country: any) => {
```

Comment:
Is there a type we can use besides `any`?

Reply:
Looks like its a loosey-goosey array we get from the library: https://github.com/jackocnr/intl-tel-input/blob/baeead220dd3462841595648fad7f5b14c383bd3/src/js/data.js

I did notice there's a types library here: https://www.npmjs.com/package/@types/intl-tel-input that we could install. Think its worth it to get around this `any`?

Reply:
Eh, I think I tried pulling that types library in for the RC branch and had trouble with it, unless you can figure it out. If it's too much trouble, we can ignore it.

Reply:
Looks like that types library doesn't even export the object that we'd use here, so its a moot point. Lets just stick with `any` for now if you don't mind?